packages/kernel/src/kernel.ts

```diff
@@ -533,14 +533,18 @@ export abstract class BaseKernel implements IKernel {
   private async _execute(msg: KernelMessage.IMessage): Promise<void> {
     const executeMsg = msg as KernelMessage.IExecuteRequestMsg;
     const content = executeMsg.content;
-    this._executionCount++;
+    if (content['store_history']) {
```

Comment:
Since `content` is well typed, something like the following should also work:

```suggestion
    if (content.store_history) {
```